packages/prisma/src/prisma/schema.prisma

```diff
@@ -87,9 +87,9 @@ model Attachment {
 
   type AttachmentType
 
-  question   Question? @relation(fields: [questionId], references: [id])
-  questionId String?   @db.Uuid
-  owner      User      @relation(fields: [ownerId], references: [id])
+  question   Question? @relation(fields: [questionId], references: [id], onDelete: SetNull, onUpdate: Cascade)
```

Comment:
Special case: if a question is deleted, we do not want to delete all of the question instances (we want to allow for this use case, hence the copying of questionData into the instance). `SetNull` will ensure that the question property on instances of a deleted question are set to `null`.